admin/services/data-access/sql.service.js

```diff
@@ -6,6 +6,20 @@ const dateService = require('../date.service')
 const poolConfig = require('../../config/sql.config')
 const moment = require('moment')
 const logger = require('../log.service').getLogger()
+const retry = require('./retry-async')
+
+const retryConfig = {
+  attempts: 3,
```

Comment:
we should make these overridable via env vars so we can tweak during load tests.

Reply:
Done

Reply:
Done